src/pages/self-assessment/self-assessment.scss

```diff
@@ -1,4 +1,11 @@
 page-self-assessment {
+    h1 {
```

Comment:
Could we use the `text-center` attribute for the h1 instead of making a one off style?